### PR TITLE
Documentation of common data

### PR DESCRIPTION
**Pull request by jacoterh**
Each commondata file now comes with the following info:

- doi
- location: where the data can be found in the paper
- arxiv: arXiv number 
- hepdata: link to hepdata entry
- units
- description
- luminosity (in fb^-1)

### CODE REVIEW

File: commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml
Comment (jacoterh) [inline commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml:1]: Was previously CMS_t_tch_13TeV_diff_Yt

File: commondata/CMS_t_tch_13TeV_2019_diff_Yt.yaml
Comment (jacoterh) [inline commondata/CMS_t_tch_13TeV_2019_diff_Yt.yaml:1]: Was previously CMS_t_tch_13TeV_2016_diff_Yt

**Comment by giacomomagni** [comment]
Hi @jacoterh (and anyone else that have contributed to this effort). 
This is really an impressive amount of work 😍 😍 😍

File: commondata/CMS_t_tch_13TeV_2019_diff_Yt.yaml
Comment (giacomomagni) [reply commondata/CMS_t_tch_13TeV_2019_diff_Yt.yaml:1]: okay, did you renamed also the corresponding theory files?
(same for above)

File: commondata/ATLAS_ggF_13TeV_2015.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
-dataset_name: ATLAS_H_13TeV_2015_pTH
+dataset_name: ATLAS_ggF_13TeV_2015
```
Comment (giacomomagni) [inline commondata/ATLAS_ggF_13TeV_2015.yaml:1]: does this mean that the dataset was mismatched?

**Comment by LucaMantani** [comment]
> Hi @jacoterh and @LucaMantani (and anyone else that have contributed to this effort). This is really an impressive amount of work 😍 😍 😍

It was all Jaco, indeed it's really nice!! :)

File: commondata/CMS_t_tch_13TeV_2019_diff_Yt.yaml
Comment (jacoterh) [reply commondata/CMS_t_tch_13TeV_2019_diff_Yt.yaml:1]: Fixed in 2c32c9e

File: commondata/ATLAS_ggF_13TeV_2015.yaml (hunk 1 above)
Comment (jacoterh) [reply commondata/ATLAS_ggF_13TeV_2015.yaml:1]: The numbers in this yaml file correspond to Fig. 2a in [ATLAS_ggF_13TeV_2015](https://arxiv.org/pdf/1805.10197.pdf), so it was not labelled correctly before. The smefit2.0 paper also writes [ATLAS_H_13TeV_2015_pTH](http://cdsweb.cern.ch/record/2682844/files/ATLAS-CONF-2019-032.pdf) in Table 5.2, but this should be ATLAS_ggF_13TeV_2015 if you look for instance at the number of datapoints. We never included ATLAS_H_13TeV_2015_pTH in smefit2.0.

File: commondata/CMS_t_sch_8TeV.yaml [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+arxiv: 1603.02555
+hepdata: none
+units: pb
+description: single top quark production cross-section in the s-channel
+luminosity: 19.7
 num_data: 1
 num_sys: 1
 data_central: 13.4
-statistical_error: 1.34
+statistical_error: 0
```
Comment (giacomomagni) [inline commondata/CMS_t_sch_8TeV.yaml:12]: was this a bug in the end?

**Review by giacomomagni — COMMENTED** [review]
if you upload the predictions also for the other 2 Atlas datasets I think we are all good!

File: commondata/CMS_t_sch_8TeV.yaml (hunk 2 above)
Comment (jacoterh) [reply commondata/CMS_t_sch_8TeV.yaml:12]: Yes it was! If you check Sec. 9 (combined result) in https://arxiv.org/pdf/1603.02555.pdf, you can see that it is correct this way.

File: commondata/ATLAS_t_tch_13TeV_inc.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+dataset_name: ATLAS_t_tch_13TeV
```
Comment (giacomomagni) [inline commondata/ATLAS_t_tch_13TeV_inc.yaml:1]:
```suggestion
dataset_name: ATLAS_t_tch_13TeV_inc
```

File: commondata/ATLAS_t_tch_8TeV_diff_Yt.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+dataset_name: ATLAS_t_tch_8TeV
```
Comment (giacomomagni) [inline commondata/ATLAS_t_tch_8TeV_diff_Yt.yaml:1]:
```suggestion
dataset_name: ATLAS_t_tch_8TeV_diff_Yt
```

File: commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml [hunk 5]
```diff
@@ -1,151 +1,35 @@
 dataset_name: CMS_t_tch_13TeV_2016_diff_Yt
-num_data: 5
-num_sys: 19
+doi: none
+location: Figure 5 (right panel), (normalised -> absolute)
+arxiv: https://inspirehep.net/literature/1456074
+hepdata: none
+units: pb
+description: single-top-quark production in the t-channel differential in the rapidity of the top + anti-top
```
Comment (giacomomagni) [inline commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml:7]: is this description correct? It sounds more tt bar ...

File: commondata/ATLAS_CMS_SSinc_RunI.yaml [hunk 6]
```diff
@@ -1,4 +1,12 @@
 dataset_name: ATLAS_CMS_SSinc_RunI
+doi: 10.17182/hepdata.78403.v1/t2
+location: Data from Tab 8, Fig 7 (first 20), Data from Fig 1 (last two)
+arxiv: 1606.02266, 1507.04548
+hepdata: ['https://www.hepdata.net/record/ins1468068?version=1&table=Table%202', '']
```
Comment (giacomomagni) [inline commondata/ATLAS_CMS_SSinc_RunI.yaml:5]:
```suggestion
hepdata: https://www.hepdata.net/record/ins1468068?version=1&table=Table%202
```

File: commondata/ATLAS_tt_AC_13TeV.yaml [hunk 7]
```diff
@@ -1,4 +1,11 @@
 dataset_name: ATLAS_tt_AC_13TeV
+doi: done
+location: Table 2
+arxiv: https://inspirehep.net/literature/1743677
+hepdata: none
+units: none
+description: statistical error refers to total uncertainty
```
Comment (giacomomagni) [inline commondata/ATLAS_tt_AC_13TeV.yaml:7]:
```suggestion
description: top-anti-top Charge Asymmetry, statistical error refers to total uncertainty
```

File: commondata/CMS_tttt_13TeV.yaml [hunk 8]
```diff
@@ -1,4 +1,11 @@
 dataset_name: CMS_tttt_13TeV
+doi: none
+location: abstract
+arxiv: 1710.10614
+hepdata: none
+units: pb
+description: Four top quark prediction
```
Comment (giacomomagni) [inline commondata/CMS_tttt_13TeV.yaml:7]:
```suggestion
description: Four top quark inclusive cross section
```
?

File: commondata/CMS_tttt_13TeV_run2.yaml [hunk 9]
```diff
@@ -1,4 +1,11 @@
 dataset_name: CMS_tttt_13TeV_run2
+doi: none
+location: abstract
+arxiv: 1908.06463
+hepdata: none
+units: pb
+description: four top quark production
```
Comment (giacomomagni) [inline commondata/CMS_tttt_13TeV_run2.yaml:7]:
```suggestion
description: four top quark inclusive cross section
```

File: commondata/LEP_eeWW_182GeV.yaml [hunk 10]
```diff
@@ -1,4 +1,11 @@
 dataset_name: LEP_eeWW_182GeV
+doi: 10.1016/j.physrep.2013.07.004
+location: Tab. 5.6
+arxiv: 1302.3415
+hepdata: none
+units: pb
+description: Combined W− differential angular cross-section in the 10 angular bins
+luminosity: 0.16390 # fb^-1
```
Comment (giacomomagni) [inline commondata/LEP_eeWW_182GeV.yaml:8]:
```suggestion
luminosity: 0.16390 # pb^-1
```

File: commondata/LEP_eeWW_189GeV.yaml [hunk 11]
```diff
@@ -1,4 +1,11 @@
 dataset_name: LEP_eeWW_189GeV
+doi: 10.1016/j.physrep.2013.07.004
+location: Tab. 5.6
+arxiv: 1302.3415
+hepdata: none
+units: pb
+description: Combined W− differential angular cross-section in the 10 angular bins
+luminosity: 0.58795 # fb^-1
```
Comment (giacomomagni) [inline commondata/LEP_eeWW_189GeV.yaml:8]:
```suggestion
luminosity: 0.58795 # pb^-1
```

File: commondata/LEP_eeWW_206GeV.yaml [hunk 12]
```diff
@@ -1,4 +1,11 @@
 dataset_name: LEP_eeWW_206GeV
+doi: 10.1016/j.physrep.2013.07.004
+location: Tab. 5.6
+arxiv: 1302.3415
+hepdata: none
+units: pb
+description: Combined W− differential angular cross-section in the 10 angular bins
+luminosity: 0.63051 # fb^-1
```
Comment (giacomomagni) [inline commondata/LEP_eeWW_206GeV.yaml:8]:
```suggestion
luminosity: 0.63051 # pb^-1
```

File: commondata/LEP_eeWW_198GeV.yaml [hunk 13]
```diff
@@ -1,4 +1,11 @@
 dataset_name: LEP_eeWW_198GeV
+doi: 10.1016/j.physrep.2013.07.004
+location: Tab. 5.6
+arxiv: 1302.3415
+hepdata: none
+units: pb
+description: Combined W− differential angular cross-section in the 10 angular bins
+luminosity: 0.60505 # fb^-1
```
Comment (giacomomagni) [inline commondata/LEP_eeWW_198GeV.yaml:8]:
```suggestion
luminosity: 0.60505 # pb^-1
```

**Review by giacomomagni — COMMENTED** [review]
LGTM
I only noticed some small typos.

File: commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml (hunk 5 above)
Comment (jacoterh) [reply commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml:7]: top + anti-top refers to the sum of single top and single anti-top production

Comment (jacoterh) [inline commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml:7]:
```suggestion
description: single-top-quark production in the t-channel differential in the rapidity of either the top or the anti-top
```

File: commondata/LEP_eeWW_182GeV.yaml (hunk 10 above)
Comment (jacoterh) [reply commondata/LEP_eeWW_182GeV.yaml:8]: 163.90 pb^-1 = 0.16390 fb^-1 and the units of the lumi entry are fb^-1 everywhere

File: commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml (hunk 5 above)
Comment (giacomomagni) [reply commondata/CMS_t_tch_13TeV_2016_diff_Yt.yaml]: okay, thanks for the clarification

**Pull request merged**